utils/test/test_config.py

```diff
@@ -0,0 +1,106 @@
+from collectdtesting.utils import ParsedConfig
```

Comment:
No links between the util package and collectdtesting.  Move anything from the testing package to utils if needed by collectd plugin code or unit tests.